cypress/integration/seededFlows/adminFlows/config/authenticationSection.spec.js

```diff
@@ -41,7 +41,9 @@ describe('Authentication Section', () => {
             );
           });
 
-          cy.url().should('contains', '/admin/customization/config');
+          // The next comment suggests this had reloaded the page
+          // instead of just showing the snackbar before.
+          cy.visit('/admin/customization/config');
```

Comment:
```suggestion
          // The page doesn't automatically reload on submission, 
          // so we reload manually to check the settings have been persisted
          cy.reload();
```
Just a suggestion - `cy.reload()` will do the same thing as this `visit` but I just enjoy that `reload` describes the action we're taking a little better 🤷‍♀️ 

> The customization submission no longer reloads the page (did it once? it looks like it might have from the test comments) 

Yeah you're right, I think it used to reload the page, then there was some refactoring so that all of these config flows happen async without page reload. I wonder if there are more of these `adminFlows/config` specs that might need similar updates 🤔

Reply:
I'll run the rest of the admin flows in chrome to check - should be a relaxing morning watching the robot click buttons for me